.github/CODEOWNERS

```diff
@@ -354,11 +361,11 @@
 /pkg/serializer/internal/metrics/origin_mapping.go  @DataDog/agent-processing-and-routing @DataDog/agent-integrations
 /pkg/serverless/                        @DataDog/serverless @Datadog/serverless-aws
 /pkg/serverless/appsec/                 @DataDog/asm-go
-/pkg/status/                            @DataDog/agent-shared-components
-/pkg/telemetry/                         @DataDog/agent-shared-components
+/pkg/status/                            @DataDog/agent-runtimes
```

Comment:
Is the status owned by Runtimes ? I thought it would go with diagnose and flare, ie. user interactions and debugging

Reply:
I'd +1 that this goes to agent-configuration team as well - it's more closely tied with the configuration aspects than "building blocks" devx.